src/nested_pandas/series/accessor.py

```diff
@@ -114,6 +114,11 @@ def to_flat(self, fields: list[str] | None = None) -> pd.DataFrame:
 
         return pd.DataFrame(flat_series)
 
+    @property
+    def list_lengths(self) -> np.ndarray:
```

Comment:
This says it returns a numpy array, but it looks like the property it references is a `list[int]`, unless I'm missing something

Reply:
To be clear, I think that `list[int]` is the right return type, and this type marker should change to match

Reply:
Nice catch!